terminus-ui/date-range/src/date-range.component.ts

```diff
@@ -7,6 +7,7 @@ import {
   OnInit,
   Output,
   ViewEncapsulation,
+  ChangeDetectorRef,
```

Comment:
We've started alphabetizing multi-imports. Looks like there is a vs code plugin that might be able to make it easy